parseIptunData caused `index out of range` panic

The `ip link` command dump:
```
13: tunl0@NONE: <NOARP> mtu 1480 qdisc noop state DOWN mode DEFAULT group default qlen 1000
    link/ipip 0.0.0.0 brd 0.0.0.0
14: ipip0@NONE: <NOARP,UP,LOWER_UP> mtu 1480 qdisc noqueue state UNKNOWN mode DEFAULT group default qlen 1000
    link/ipip 0.0.0.0 brd 0.0.0.0
15: ip6tnl0@NONE: <NOARP> mtu 1452 qdisc noop state DOWN mode DEFAULT group default qlen 1000
    link/tunnel6 :: brd ::
16: ipip60@NONE: <NOARP,UP,LOWER_UP> mtu 1452 qdisc noqueue state UNKNOWN mode DEFAULT group default qlen 1000
    link/tunnel6 :: brd ::
```

The panic backtrace are:
```
panic: runtime error: index out of range

 github.com/vishvananda/netlink.parseIptunData(0xcc4b40, 0xc000214580, 0xc00028ee00, 0xd, 0x10)
         /go/src/github.com/vishvananda/netlink/link_linux.go:2580 +0x2c1
 github.com/vishvananda/netlink.LinkDeserialize(0x0, 0xc00028497c, 0x590, 0x113c, 0x9, 0xc00028c700, 0x8, 0x10)
         /go/src/github.com/vishvananda/netlink/link_linux.go:1608 +0x1482
 github.com/vishvananda/netlink.(*Handle).LinkList(0x1359fd0, 0xbff8b3, 0x9, 0x21a, 0xc000034f70, 0xc000034f00)
         /go/src/github.com/vishvananda/netlink/link_linux.go:1813 +0x2e6
 github.com/vishvananda/netlink.LinkList(...)
         /go/src/github.com/vishvananda/netlink/link_linux.go:1791

```

--- a/link_linux.go
+++ b/link_linux.go
@@ -2116,7 +2116,11 @@ func parseVxlanData(link Link, data []syscall.NetlinkRouteAttr) {
 		case nl.IFLA_VXLAN_GBP:
 			vxlan.GBP = true
 		case nl.IFLA_VXLAN_FLOWBASED:
-			vxlan.FlowBased = int8(datum.Value[0]) != 0
+			// NOTE(vish): Apparently this message can be sent with no value.
+			//             Unclear if the others  be sent that way as well.
+			if len(datum.Value) > 0 {
+				vxlan.FlowBased = int8(datum.Value[0]) != 0
+			}
 		case nl.IFLA_VXLAN_AGEING:
 			vxlan.Age = int(native.Uint32(datum.Value[0:4]))
 			vxlan.NoAge = vxlan.Age == 0

